scripts/convert_original_stable_diffusion_to_diffusers.py

```diff
@@ -148,3 +149,9 @@
         pipe.controlnet.save_pretrained(args.dump_path, safe_serialization=args.to_safetensors)
     else:
         pipe.save_pretrained(args.dump_path, safe_serialization=args.to_safetensors)
+
+    if args.half:
+        # convert pipe to half
+        pipe.unet = pipe.unet.half()
+        pipe.vae = pipe.vae.half()
+        pipe.text_encoder = pipe.text_encoder.half()
```

Comment:
This should probably be done before saving the pipeline no?

Reply:
my bad, thank you!